spec/pluto/operation/pad_spec.cr

```diff
@@ -0,0 +1,53 @@
+require "../../spec_helper"
+
+describe Pluto::Operation::Pad do
+  describe "#pad" do
+    it "works with ImageGA" do
+      image = ga_sample
+      image.crop!(0, 0, image.width // 2, image.height)
```

Comment:
Ha, because the sample image is already surrounded by black, I had to crop it in half so that the `repeat` pixels weren't just repeating black and yielding the same hash as the black padded image.

Reply:
What do you think about opening a separate PR where we replace this image with another one? We can merge that one, then update this one.

Reply:
Might just be a regular commit as well, no need for an additional review.

Reply:
Want me to look for and add another sample image for `pluto_samples` ? Can do, just making sure that was what you were suggesting :) Would save a crop operation in this spec at least.

Reply:
I would replace the existing image.

Reply:
Replacing the existing image would cause the hashes in all of the other specs to change and fail. If that's desirable, doing it in a separate PR might be better than rolling it into this one. 

If you create an issue with the request, I can probably get to it on my own in a week or two.